### PR TITLE
Added data policy for the ios library

### PR DESCRIPTION
**Pull request by royjit**
Screen Shot 2020-12-02 at 5 21 52 PM

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/lib/menu.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
     "utilities",
     "client-configuration",
     "debugging",
+    "info",
```
Comment (drochetti) [inline docs/lib/menu.json:18]: hhhm maybe "Privacy policy" or something more explicit?

**Review by drochetti — APPROVED** [review]
This is great, thanks!

I suggested a different menu title, but will approve and leave the decision to you.

**Review by jamesonwilliams — APPROVED** [review]

File: docs/lib/menu.json (hunk 1 above)
Comment (palpatim) [reply docs/lib/menu.json:18]: Nit: This is not a policy, this is a description.

File: docs/lib/info/fragments/native_common/data-information/common.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
+Apple App Store require the developer to provide the data usage policy of the app when they submit the app to the store. Given below are the different categories identified by Apple and the corresponding data type used by the Amplify library.
```
Comment (palpatim) [inline docs/lib/info/fragments/native_common/data-information/common.md:1]:
```suggestion
Apple requires app developers to provide the data usage policy of the app when they submit their app to the App Store. Below are the different categories identified by Apple and the corresponding data type used by the Amplify library.
```

**Review by palpatim — APPROVED** [review]
approved w/changes

File: docs/lib/menu.json (hunk 1 above)
Comment (royjit) [reply docs/lib/menu.json:18]: Keeping with Info for now. Will update this later if required.

**Pull request merged**